### DISCUSSION TITLE
Welcome to kool Discussions!

### DISCUSSION THREAD

**Discussion by fabmax** [fabmax/kool#34] [Announcements]
## 👋 Welcome to kool Discussions!
  This is a place to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by Protoxy22** [comment]
It's very kool here ! Great idea

**Comment by jfelrod1960** [comment]
Hey @fabmax!  Where did you get your documentation concerning a physics time stepper to run in each time step?  I'm trying to build a physics world for my RTS game.  Thanks in advance!  Jeff

**Reply by fabmax** [reply]
Hi Jeff!

The PhysX documentation is a good starting point. In particular the [Simulation loop](https://nvidia-omniverse.github.io/PhysX/physx/5.6.0/docs/Simulation.html#the-simulation-loop) and [Substepping](https://nvidia-omniverse.github.io/PhysX/physx/5.6.0/docs/Simulation.html#substepping) chapters might be helpful. But in general the PhysX docs offer a lot of information regarding physics stuff.

**Reply by jfelrod1960** [reply]
Thank you!
